rst_files/getting_started.rst

```diff
@@ -336,38 +303,6 @@ which provides a static HTML representations of notebooks
 
 QuantEcon also hosts the `QuantEcon Notes <http://notes.quantecon.org/>`_ website, where you can upload and share your notebooks with other economists and the QuantEcon community
 
-.. _julia_repl: 
-
-The REPL
```

Comment:
I didn't really see the need for this section... it seemed a bit redundant and possibly to much info for getting started

Reply:
Normally I prefer to strip things out of the introduction.  The problem is that without the REPL section, I think people will be very confused if they need to do package operations or people read about the REPL online.

One option is to tag it as `(Advanced)` in the title i.e. `The REPL (Advanced)` or something like that.   Then readers know it is somewhat optional.

Otherwise, there is no where else they would see the REPL unless they read the (optional) advanced julia section.

Reply:
But I think what is in there could be simplified a bit bit.

Reply:
I thought the REPL was mentioned earlier when you open Julia and switch the package manager

I also added earlier that you can open julia by typing `julia` in the REPL

Reply:
It was used in the setup instructions, but was a mechanical step that they may have thought was a one-time thing to do.  It wasn't described as something that they would use otherwise, nor was it explained how to access it from a jupyterhub installation.  In particular, we want them to associate package operations with using the REPL

As I said, though, it could potentially be condensed.

Reply:
I think we should explain in the setup - it can be discouraging to just be entering commands without knowing what's going on, so I think we should describe the REPL and package manager when they're needed

Reply:
Regarding the REPL section, we don't need to tell people how to open the REPL, given that we already assume they can open it at the top of the lecture.  I suggest cutting those instructions.

On the topic of making this lecture more friendly, I recommend a few more sentences after ``] add IJulia InstantiateFromURL; precompile`` 

I hate typing something into my fresh clean installation when I don't know what's going on...

Just a couple of dot points would be great, saying that

1. `]` activates the package manager 
2. Julia needs to install additional code in order to run the programs in the lectures
3. This code comes in libraries called packages, about which we will say more below
4. The QE team has organized a collection of useful packages, and these instructions install them in one go
5. You can find the list of packages installed by this command by typing ??

Reply:
The problem is that many people (i.e. any students I have or the schools we suggest setup jupyterhub) will not actually run through that setup process.  It is only for the desktop environment and not docker or jupyterhub.  Launching the REPL has different steps in a jupyterhub installation.

We can put a few more sentences if you like.  I have the opposite preference for setup instructions (i.e., I like to be told exactly what to do with little exposition, and blindly follow instructions) but you have a defensible position.  I don't think it causes much damage here either way.

For your text, another place your text could work is when we use the `InstantiateFromURL` package (which happens for desktop, jupyterhub, etc.).  At that point in the instructions, the `add IJulia` is basically just to get Jupyter up and running in a minimal setup step, and the InstantiateFromURL is used to get the packages.  Having the extra discussion in the setup might be confusing since the whole `IJulia` step is not quantecon specific.

Another approach is to have the `] add InstantiateFromURL` from within Jupyter so that the setup step prior to jupyter is just `] add IJulia; precompile`.  This may be an easier flow, doesn't break the jupyterhub setups, and your text makes a lot of sense.